src/services/infinispanRefData.ts

```diff
@@ -140,3 +140,13 @@ export enum IndexedStorage {
   persistent = 'filesystem',
   volatile = 'local_heap',
 }
+
+export enum PersistentCacheStorage {
+  FileStore = 'File Store',
+  RemoteStore = 'Remote Store',
+  SQLStore = 'SQL Store',
+  JDBCStore = 'JDBC Store',
```

Comment:
The name should be `'JDBC String Based Store'' in order to distinguish it from the `SQLStore` which is also utilises JDBC.